### DISCUSSION TITLE
Binary Size

### DISCUSSION THREAD

**Discussion by chromer030** [wezterm/wezterm#642] [General]
Thank you for developing this lovely terminal, but a question :
What causes wezterm to have 20 mb in binary size and 65 mb in insatllation size while Alacritty is just 2 MB and both are in native rust ?

**Comment by wez** [comment]
WezTerm includes a lot more features than Alacritty, including iTerm compatible image decoding of several image formats, ssh and TLS based multiplexing, and bundles copies of JetBrains Mono and Noto Color Emoji fonts.  WezTerm prefers to statically link dependencies so that it is simpler to manage them and so that we can ensure that we have the both the latest and a consistent set of versions, and a consistent default out of the box experience across platforms.

If you're on Linux, up until the WezTerm release that I pushed a few minutes ago, wezterm also statically linked against openssl.  Rust 1.51 finally allows wezterm to not statically link to it on Linux, so that should shave a little bit off the size.

**Reply by chromer030** [reply]
Perfect, Thank you for explanations and this good project.
